src/unix/fs.c

```diff
@@ -229,9 +229,46 @@ static ssize_t uv__fs_futime(uv_fs_t* req) {
 #endif
 }
 
-
```

Comment:
please leave this empty line alone, we separate functions with 2 line breaks, generally